pkg/etcdutil/etcdutil_test.go

```diff
@@ -115,10 +121,13 @@ func (t *testEtcdUtilSuite) checkMember(c *C, mid uint64, m *etcdserverpb.Member
 	c.Assert(m.PeerURLs, DeepEquals, t.urlsToStrings(cfg.APUrls))
 }
 
-func (t *testEtcdUtilSuite) TestMemberUtil(c *C) {
+func (t *testEtcdUtilSuite) TestEtcdUtil(c *C) {
 	for i := 1; i <= 3; i++ {
 		t.testMemberUtilInternal(c, i)
 	}
+
+	t.testRemoveMember(c)
```

Comment:
- in `testMemberUtilInternal`, we need to generate config and start etcd manually for testing members.
- in `testRemoveMember` and `testDoOpsInOneTxnWithRetry`, we want to use `integration.NewClusterV3` to simply the creation of etcd cluster.

but both of them will write/read the global logger, which may cause DATA RACE if running them parallel, so we let them do not run parallel here.